go/vt/sqlparser/testdata/mysql_keywords.txt

```diff
@@ -1,5 +1,5 @@
 // Code generated by `SELECT * FROM INFORMATION_SCHEMA.KEYWORDS`.
-// Reference: https://dev.mysql.com/doc/refman/8.0/en/information-schema-keywords-table.html
+// Reference: https://dev.mysql.com/doc/refman/8.4/en/information-schema-keywords-table.html
```

Comment:
Probably makes sense to use the MySQL 9.0 output. It's almost identical, except for the fact that it also has Vector. AKA, it's exactly what we're aiming to support at the moment.

Reply:
@GrahamCampbell As mentioned in #17525 I'd like to stay with this here for now and look at this separately later.